config/jobs/kubernetes/cloud-provider-gcp/cloud-provider-gcp-periodics.yaml

```diff
@@ -0,0 +1,32 @@
+periodics:
+- interval: 3h
+  cluster: k8s-infra-prow-build
+  name: ci-cloud-provider-gcp-conformance-latest
+  annotations:
+    testgrid-tab-name: Conformance - Cloud Provider GCP - master
+    description: Runs conformance tests using kubetest2 against cloud-provider-gcp master on GCE
+  labels:
+    preset-service-account: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/krte:v20200806-28035c4-1.16
```

Comment:
I'm not sure which tag of this KRTE release I should be using. [This](https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/post-test-infra-push-krte/1291480780802887680) is the postsubmit that built it.

Reply:
we need your fix so this is fine, but eventually we should use the auto generated tag.